src/libs/collect.c

```diff
@@ -116,7 +116,7 @@ static void _populate_collect_combo(GtkWidget *w);
 
 const char *name(dt_lib_module_t *self)
 {
-  return _("collect images");
+  return _("generate collection");
```

Comment:
Im' not fond of generate here. Has "dynamic collection" been considered ?

Reply:
dynamic collection is great. Or, as suggested by @quovadit, create collection

Reply:
I suggested generate as it doesn't imply permenance (create and build suggest something to be saved) but on reflection I don't like the sound of it - I'm not sure why. "create" is ok with me. I don't mind "dynamic" but if it's a term used by Adobe I'd prefer to steer clear of it (just to avoid the "Adobe clone" label).

Reply:
I did a quick search on the Web and I don't see "dynamic collection" as being used a label. Of course one can talk about dynamic collections when describing it, but the module in Lr is named "Collections". As this match my recollection.

Reply:
I prefer "create" because the word "collection" is used on its own in various places in the UI and then we have a single word to reference the output of the "create collection" module.

Reply:
"create" is fine by me too.